matchms/importing/load_from_json.py

```diff
@@ -0,0 +1,61 @@
+import ast
+import json
+import numpy
+from ..Spectrum import Spectrum
+
+
+def load_from_json(filename):
+    """Load spectrum(s) from json file.
+
```

Comment:
Explain expected format of JSON document
```suggestion

JSON document formatted like the `GNPS Spectra library <https://gnps-external.ucsd.edu/gnpslibrary>`_
```